app/main.py

```diff
@@ -1,14 +1,17 @@
 from fastapi import FastAPI
 import uvicorn
 
-from api.routers import health_check_router
+from api.routers import health_check_router, iss_position_router, login_router
 from logger_config import setup_log_config
 
 setup_log_config()
 
 app = FastAPI()
 
 app.include_router(health_check_router.router)
```

Comment:
we must add the authentication here for each for the router. We must add a dependency here and only fastapi will force user to authenticate before calling any service.

Reply:
Implemented in health check router